README.md

```diff
@@ -1,9 +1,17 @@
 FileProvider
 ==========
-AppVeyor: [![AppVeyor](https://ci.appveyor.com/api/projects/status/8pryt261xf7q0cb6/branch/dev?svg=true)](https://ci.appveyor.com/project/aspnetci/FileSystem/branch/dev)
 
-Travis:   [![Travis](https://travis-ci.org/aspnet/FileSystem.svg?branch=dev)](https://travis-ci.org/aspnet/FileSystem)
+| Build System  | Status |
+| :-----------: | :-----  |
+| AppVeyor  | [![AppVeyor](https://ci.appveyor.com/api/projects/status/8pryt261xf7q0cb6/branch/dev?svg=true)](https://ci.appveyor.com/project/aspnetci/FileSystem/branch/dev)  |
+| Travis  | [![Travis](https://travis-ci.org/aspnet/FileSystem.svg?branch=dev)](https://travis-ci.org/aspnet/FileSystem)  |
 
 File Provider abstractions. Contains file system abstractions and file system globbing.
 
 This project is part of ASP.NET Core. You can find samples, documentation and getting started instructions for ASP.NET Core at the [Home](https://github.com/aspnet/home) repo.
+
+In addition, there are some other FileProvider implementations that you may find useful:
```

Comment:
Slightly tweaked words: `In addition, there are some **community** FileProvider implementations that you may find useful:`